demisto_sdk/scripts/add_marketplace_to_schemas.py

```diff
@@ -0,0 +1,52 @@
+from copy import deepcopy
+from pathlib import Path
```

Comment:
let's talk about what ever this should be in the repo, and if so, it shall include UTs.